scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -36,8 +36,8 @@ workflow GvsBulkIngestGenomes {
         # Begin GvsImportGenomes
         File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
 
-        # set to "NONE" to ingest all the reference data into GVS for VDS (instead of VCF) output
-        String drop_state = "NONE"
+        # set to "ZERO" to drop GQ0 upon ingest into GVS for VDS (instead of VCF) output
+        String drop_state = "ZERO"
```

Comment:
Is the intended result that we change our default behavior to drop GQ0 or just to verify that it's viable to do so for Echo?

Reply:
To drop GQ0. That was my understanding of what we are doing for Echo.

Reply:
Yes, I supposed the question was whether our guiding principle is to make whatever we're doing for Echo our default behavior (for dev, beta customers, Broad internal, etc) or simply to verify that the behavior we want in Echo works and ensure tests are present to validate that when necessary.  But since defaults are easy to update anyway, it's ultimately not a big enough deal to delay merging this PR this close to the end of the sprint